src/layouts/Nav/nav.module.styl

```diff
@@ -36,14 +36,28 @@ $navbar($height, $font-size = 20px)
   padding-left $navbar-padding-vertical
   padding-right $navbar-padding-vertical
   border-bottom 1px solid $gray
+  display flex
+  justify-content space-between
```

Comment:
This class already has flex, if you want to justify the content then pass the term 'between' as the first arg to `$grid`